### ISSUE TITLE
Cannot use Storage Explorer 

### ISSUE THREAD

**Issue by FrankPan1990** [Microsoft/AzureStorageExplorer#969]
**Storage Explorer Version**: 1.6.0
**Platform/OS Version**: Windows
**Architecture**: 
**Regression From**: 

#### Steps to Reproduce: ####
1.  When I log in my account for subscriptions, the problem happened
2.  The problem happened when click load more.
3. The problem happened like "the requested action could not be completed." Details: Cannot read property 'connectionString' of undetfined. Then, the explorer unable to retrieve child resources. Details Cannot read property 'connectionString' of undefined.

EXPLORER part.
![image](https://user-images.githubusercontent.com/44986762/49559017-20ea7d00-f948-11e8-8da6-38c4e9cdd0ec.png)

ERROR message
![image](https://user-images.githubusercontent.com/44986762/49558806-58a4f500-f947-11e8-8264-8279cc01d561.png)

![image](https://user-images.githubusercontent.com/44986762/49558817-5fcc0300-f947-11e8-9929-45fd02a0934a.png)

#### Expected Experience: ####
The Azure Storage Explorer should work and load all storage account.
#### Actual Experience: ####
The Azure storage explorer cannot load all storage account under my subscription.

**Comment by bretthackermsft** [comment]
I have this same issue, and I've had it for several versions now, and nobody seems to care. The workaround, if you're interested, is to click "File - New Window".

Since this has been complained about for so long, I'm assuming it's a config corruption or something rather than an actual bug. If that's the case, then TEAM, could you please update the readme, or the doc, or the Wiki, or something, to let us know how to fix it? I've even offered to share my session with a dev so they could see the repro - no response.

**Comment by MRayermannMSFT** [comment]
Hi @bretthackermsft  and @FrankPan1990 , can y'all try something for me? Note: doing this will remove all of your attached resources, though I think it should fix your issue.

Open dev tools (f12)

1. Go to the Application tab
2. Navigate to Local Storage -> file:// (see picture)
3. Filter by CustomConnections
4. For all results, double click on the Value cell, and replace the value with []
5. Restart the app

![image](https://user-images.githubusercontent.com/21112697/49617439-6bc8bc80-f969-11e8-895a-ebc9f6fc8f04.png)

If losing all of your attached resources is a big deal for you, let me know, and we can try to only get rid of the corrupted connection.

**Comment by bretthackermsft** [comment]
!!!!

That worked, thanks very much. The string in that array was a since-removed manual connection. Since this is the fix, can't you trap that error, assuming it's throwing as you loop that array and setup each connection, and prompt to remove it? Alternatively, at least update the error message in the dialog?

**Comment by MRayermannMSFT** [comment]
Yes! We should definitely do that. I've made a new issue to track that effort. https://github.com/Microsoft/AzureStorageExplorer/issues/985

**Comment by MRayermannMSFT** [comment]
@FrankPan1990 , since the workaround unblocked Bret, I'll assume it'll work for you too so I'll close this issue. Please comment if that ends up being not the case.

**Comment by FrankPan1990** [comment]
Sorry for the delay. the problem disappear after clear custom connection. Thank you.

**Comment by rustam-umarov** [comment]
@MRayermannMSFT you're the best! Thanks!

**Comment by MRayermannMSFT** [comment]
Thanks @rustam-umarov 😄 We try really hard to be the best! Sorry for the times we don't get quite there, and thanks for being patient when that happens.
